selfdrive/manager.py

```diff
@@ -536,8 +536,8 @@ def main():
   if os.getenv("PREPAREONLY") is not None:
     spinner_proc = None
   else:
-    spinner_text = "chffrplus" if params.get("Passive")=="1" else "openpilot"
-    spinner_proc = subprocess.Popen(["./spinner", "loading %s"%spinner_text],
+    spinner_text = "chffrplus" if params.get("Passive")=="1" else "Tesla P85+"
```

Comment:
Shouldn't it just be "Tesla" for the else here?

Reply:
Woops that was not supposed to be part of the PR. :/ That is solely for myself. It just needs to be openpilot for all others.